src/sentry/api/endpoints/organization_incident_details.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.incident import IncidentPermission
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.incidents.models import Incident
+
+
+class OrganizationIncidentDetailsEndpoint(OrganizationEndpoint):
+    permission_classes = (IncidentPermission, )
+
+    def convert_args(self, request, incident_id, *args, **kwargs):
+        args, kwargs = super(OrganizationIncidentDetailsEndpoint, self).convert_args(
+            request,
+            *args,
+            **kwargs
+        )
+        organization = kwargs['organization']
+
+        if not features.has('organizations:incidents', organization, actor=request.user):
+            raise ResourceDoesNotExist
+
+        try:
+            kwargs['incident'] = Incident.objects.get(
+                organization=organization,
+                id=incident_id,
```

Comment:
We're not using the `identifier` field?

Reply:
I think I'd originally assumed (maybe incorrectly) that field existed to avoid exposing database IDs to end users (although we actually do for most other resources....)

Reply:
I think the only reason for the `identifier` field is so that we can show this number in the ui: 
![n6iHuFEA](https://user-images.githubusercontent.com/6288560/57409884-4224f300-719e-11e9-8ce1-8678b8759766.png)

Reply:
Can't that just be `id` though? Wouldn't we want to match that UI with the number param in the URL? I think it'd be nice to avoid exposing 2 different ID-like numbers for an incident to users if we can avoid it

Reply:
@saragilford Do you have any thoughts on this? Do you think we also need the org-scoped `identifier` number?

Reply:
I like having an org-scoped identifier, so orgs don't end up with Incident #1859435789 if it's their first incident. Is there a way to include the identifier in the URL instead of the db id?

Reply:
@saragilford @lynnagara We can change our apis to use the `identifier` instead if we prefer. It's not a huge change, just need to coordinate with anything @billyvg is working on that relies on using the id.